### PR TITLE
Add the packaging metadata to build the rollbar-agent snap

### PR DESCRIPTION
**Pull request by elopio**
This is a package for the secure installation of apps that works in most Linux distributions.
Landing it upstream will enable builds that then can be distributed to many users through the Ubuntu store.

### CODE REVIEW

**Comment by elopio** [comment]
You can find more information about snapcraft here: http://snapcraft.io/

To test it in an Ubuntu 16.04 machine:

    $ sudo apt install git snapcraft
    $ git clone https://github.com/elopio/rollbar-agent
    $ cd rollbar-agent
    $ git checkout snapcraft
    $ snapcraft
    $ sudo snap install *.snap --dangerous --classic

If you have any questions or comments, please let me know.

**Comment by rivkahstandig3636** [comment]
@rokob Can you review this PR please?

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+apps:
+  rollbar-agent:
+    command: rollbar-agent
+
+parts:
+  rollbar-agent:
+    source: .
+    plugin: python
+    python-version: python2
```
Comment (rokob) [inline snap/snapcraft.yaml]: We should add `requirements: requirements.txt` also I think right?

**Comment by rokob** [comment]
LGTM, although not sure if the `requirements: requirements.txt` option is necessary

**Pull request merged**